### ISSUE TITLE
Fonts not syncing

### ISSUE THREAD

**Issue by zhwwng** [pourmand1376/obsidian-custom-font#4]
I'm not sure if it's a bug or intended, but if I put `fonts` folder in `.obsidian`, it will not be syncing at all.

**Comment by pourmand1376** [comment]
How do you sync your files? Are you using obsidian sync?

**Comment by w-zhiwei** [comment]
> How do you sync your files? Are you using obsidian sync?

Yes, im using the officient sync. My current workaround is mannually set the value of `font_folder_path` in the source code.

**Comment by pourmand1376** [comment]
Hmm, What path did you choose to fix the issue?

**Comment by w-zhiwei** [comment]
> Hmm, This is something that I haven't considered.
> 
> What path did you set to fix the issue?

I just set `const font_folder_path = "fonts"` to put the fonts folder outside `.obsidian`. I think maybe there could be an option for users to set the font folder dir themselves.

**Comment by pourmand1376** [comment]
> > Hmm, This is something that I haven't considered.
> > What path did you set to fix the issue?
> 
> I just set `const font_folder_path = "fonts"` to put the fonts folder outside `.obsidian`. I think maybe there could be an option for users to set the font folder dir themselves.

This could be a good fix. Thanks for submitting the issue.

**Comment by diatche** [comment]
I'm having a related issue, where on iOS the fonts folder is not being found even though it's there in the file tree (visible via Working Copy). On iOS, I've configured Obsidian to use `.obsidian.mobile` instead of `.obsidian`, which might be causing this.

**Comment by pourmand1376** [comment]
Currently, I put the font files inside `.obsidian/fonts`. I think this would be resolved if I make this path customizable. 

I would add that functionality as soon as I can. 

Also, I would appreciate it if anybody could make a PR.

**Comment by pourmand1376** [comment]
New Version is released [here](https://github.com/pourmand1376/obsidian-custom-font/releases/tag/1.4.1-alpha.1).

<img width="793" alt="image" src="https://github.com/pourmand1376/obsidian-custom-font/assets/32064808/10abf0cb-df69-470f-bc60-a5712ce6fcd1">

You can test it by downloading `main.js` file and replacing with current `main.js` file in `plugins/custom-fonts-loader` folder.

**Comment by pourmand1376** [comment]
This issue should be solved by now. Just change the plugin font path to something else and It would automatically sync with whatever client you have! 

If you have anymore questions, don't hesitate to ask here.
